common/common.go

```diff
@@ -112,3 +134,123 @@ func checkAllPodsSuccessfulPhase(wcClient *client.Client) func() error {
 		return nil
 	}
 }
+
+func createPodWithPVC(wcClient *client.Client) func() error {
```

Comment:
You might want to take a look at how Mario has started structuring the code in https://github.com/giantswarm/cluster-test-suites/pull/26. It might be best to pull these PVC tests out in a similar way.

Reply:
Thanks, I'll have a look

Reply:
I think it would make sense to merge https://github.com/giantswarm/cluster-test-suites/pull/26 first. Afterwards I'll rebase my PR 🙂

Reply:
I agree! (cc @mnitchev)